quartz/components/Search.tsx

```diff
@@ -18,7 +18,7 @@ export default ((userOpts?: Partial<SearchOptions>) => {
     const opts = { ...defaultOptions, ...userOpts }
     const searchPlaceholder = i18n(cfg.locale).components.search.searchBarPlaceholder
     return (
-      <div class={classNames(displayClass, "search")}>
+      <search class={classNames(displayClass, "search")}>
```

Comment:
I think the default container should be div, and diff search-container to <search> tag

```tsx
        <search id="search-container">
          <form id="search-space">
            <input
              autocomplete="off"
              id="search-bar"
              name="search"
              type="text"
              aria-label={searchPlaceholder}
              placeholder={searchPlaceholder}
            />
          </form>
          <output id="search-layout" data-preview={opts.enablePreview}></output>
        </search>
```

might have to update scss as well

Reply:
> This example demonstrates potential DOM content when dynamically including JavaScript search functionality in a web application. When search functionality is implemented entirely with JavaScript, if no form is submitted, neither a [<form>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) element nor a submit [<button>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) is required. For semantics, the <search> element is included to contain the search and filtering capabilities.

(c) https://developer.mozilla.org/en-US/docs/Web/HTML/Element/search#web_app_search

Reply:
sg, can you then also update the search-container to section and output and form accordingly?

Reply:
no, you have a js search and no form. MDN specifies for such case, that form does not required.

Reply:
> The <search> element semantically identifies the purpose of the element's contents as having search or filtering capabilities. The search or filtering functionality can be for the website or application, the current web page or document, or the entire Internet or subsection thereof.

From their example

```html
<search>
  <label>
    Find and filter your query
    <input type="search" id="query" />
  </label>
  <label>
    <input type="checkbox" id="exact-only" />
    Exact matches only
  </label>

  <section>
    <h3>Results:</h3>
    <ul id="results">
      <!-- search result content -->
    </ul>
    <output id="no-results">
      <!-- no results content -->
    </output>
  </section>
</search>
```

Ig we don't need form, but for the layout seems to align with their example